src/Views/Touchpad.vala

```diff
@@ -136,6 +147,14 @@ public class MouseTouchpad.TouchpadView : Gtk.Grid {
             click_method_transform_func
         );
 
+        disable_with_mouse_switch.notify["active"].connect (() => {
+            if (disable_with_mouse_switch.active) {
+                glib_settings.set_enum ("send-events", 2);
```

Comment:
Not a fan of these magic numbers for the enums. I was part way through  writing a branch when you opened this PR. I do some probably overly complicated binding transforms, so I guess you could either take code from here or maybe `set_string` works instead:
https://github.com/elementary/switchboard-plug-mouse-touchpad/commit/78adef2bde637f5ca097913899fcbb26ac3cc870